core/scripts/vrfv2plus/testnet/v2plusscripts/super_scripts.go

```diff
@@ -628,23 +643,23 @@ func VRFV2PlusDeployUniverse(e helpers.Environment,
 	SetCoordinatorConfig(
 		e,
 		*coordinator,
-		uint16(*coordinatorConfig.MinConfs),
-		uint32(*coordinatorConfig.MaxGasLimit),
-		uint32(*coordinatorConfig.StalenessSeconds),
-		uint32(*coordinatorConfig.GasAfterPayment),
+		uint16(coordinatorConfig.MinConfs),
+		uint32(coordinatorConfig.MaxGasLimit),
+		uint32(coordinatorConfig.StalenessSeconds),
+		uint32(coordinatorConfig.GasAfterPayment),
 		coordinatorConfig.FallbackWeiPerUnitLink,
 		coordinatorConfig.FeeConfig,
 	)
 
 	fmt.Println("\nConfig set, getting current config from deployed contract...")
 	PrintCoordinatorConfig(coordinator)
 
-	if len(*registerKeyUncompressedPubKey) > 0 {
+	if len(vrfKeyRegistrationConfig.VRFKeyUncompressedPubKey) > 0 {
 		fmt.Println("\nRegistering proving key...")
 
 		//NOTE - register proving key against EOA account, and not against Oracle's sending address in other to be able
 		// easily withdraw funds from Coordinator contract back to EOA account
-		RegisterCoordinatorProvingKey(e, *coordinator, *registerKeyUncompressedPubKey, e.Owner.From.String())
+		RegisterCoordinatorProvingKey(e, *coordinator, vrfKeyRegistrationConfig.VRFKeyUncompressedPubKey, e.Owner.From.String())
```

Comment:
> RegisterCoordinatorProvingKey(e, *coordinator, vrfKeyRegistrationConfig.VRFKeyUncompressedPubKey, e.Owner.From.String())

Is this intended to be 

> RegisterCoordinatorProvingKey(e, *coordinator, vrfKeyRegistrationConfig.VRFKeyUncompressedPubKey, vrfKeyRegistrationConfig.RegisterAgainstAddress)

Reply:
ahh, thanks a ton @kidambisrinivas ! fixed